account_move_tier_validation/models/account_move.py

```diff
@@ -28,7 +28,16 @@ def _get_validation_exceptions(self, extra_domain=None, add_base_exceptions=True
         # otherwise editing manually the values on lines dirties the field at onchange
         # since it's not in readonly because readonly="not(review_ids)", it's then
         # sent at save, and will override the values set by the user
-        return res + ["amount_total"]
+        # The other exclusions are needed to be able to generate the pdf
+        # and send the invoice by email
+        am_exceptions = [
+            "amount_total",
+            "needed_terms_dirty",
+            "is_manually_modified",
+            "is_move_sent",
+            "sending_data",
+        ]
```

Comment:
```suggestion
            "matched_payment_ids",
            "payment_state",
        ]
```

I think these should also directly be integrated. Since validation is before post (and even if validation was used after post to block payments), payment will necessarily be registered after, so these fields should never be blocking.

Reply:
Could be :) I'll take your word for it and commit this suggestion :+1: